src/chrome/content/rules/HurricaneElectric.xml

```diff
@@ -3,41 +3,27 @@
 
 		- TunnelBroker.xml
 
-
-
-	Nonfunctional subdomains:
-
-		- bgp.he.net	(refused)
-		- man.he.net	(redirects to www.he.net, valid cert)
-
-
-	bgp.he.net and faq.he.net don't support SSL at the moment
-
-
-	Problematic domains:
-
-		- (www.)hu.com *
-		- hurricaneelectric.com *
-
-	* Alternative domains, same content, using he.net certificate
-
+	Non-functional hosts
+		SSL connect error:
+		- man.he.net
+
+		SSL peer certificate was not OK:
+		- he.com
+		- www.he.com
+		- hurricaneelectric.net
 -->
-<ruleset name="Hurricane Electric">
-
-	<target host="he.com" />
-	<target host="www.he.com" />
+<ruleset name="Hurricane Electric (partial)">
 	<target host="he.net" />
-	<target host="*.he.net" />
-	<target host="hurricaneelectric.net" />
-
-
-	<securecookie host="^(?:.*\.)?he\.net$" name=".+" />
-
-
-	<rule from="^http://(?:www\.)?h(?:e\.com|urricaneelectric\.net)/"
-		to="https://www.he.net/" />
```

Comment:
Shouldn't we keep a rule like this?

Reply:
no, I missed this rule when I rewrote the ruleset. thanks.